vcn_gateways.tf

```diff
@@ -301,6 +301,97 @@ resource "oci_core_route_table" "nat" {
   count = var.create_nat_gateway ? 1 : 0
 }
 
+resource "oci_core_route_table" "igw_ngw_mixed_route_id" {
+  compartment_id = var.compartment_id
+  display_name   = var.igw_ngw_mixed_route_table_display_name
+
+  freeform_tags = var.freeform_tags
+  defined_tags  = var.defined_tags
+
+  route_rules {
+    # * With this route table, NAT Gateway is always declared as the default gateway for IPv4
+    destination       = local.anywhere
+    destination_type  = "CIDR_BLOCK"
+    network_entity_id = oci_core_nat_gateway.nat_gateway[0].id
+    description       = "Terraformed - Auto-generated at NAT Gateway creation: NAT Gateway as default gateway for IPv4"
```

Comment:
In description, keep only "NAT Gateway as default gateway for IPv4", drop the rest